test/gui/shared/scripts/helpers/WinPipeHelper.py

```diff
@@ -0,0 +1,133 @@
+import os
```

Comment:
@TheOneRing Could you review this PR? Specially this file

Reply:
Looks good.